.github/workflows/ci-front.yaml

```diff
@@ -139,4 +139,7 @@ jobs:
         uses: ./.github/workflows/actions/nx-affected
         with:
           tag: scope:frontend
-          tasks: ${{ matrix.task }}
\ No newline at end of file
+          tasks: ${{ matrix.task }}
+      - name: TypeScript Type Check
```

Comment:
That's not necessary we already have a CI task for that

Reply:
Ok